source/includes/markdown/_09-rich-text.html.md

```diff
@@ -324,6 +345,27 @@ To write rich text that contains a new external media embed:
 1. Create URL attachment with a call to [`POST /attachments`](/docs/upload-an-attachment), with `{ ..., "resource_subtype": "external", "url": "<your_url>" }`. **Important:** Use the URL that would appear in the browser address bar (e.g. `https://youtube.com/watch?v=...`), NOT the embeddable URL (e.g. `https://youtube.com/embed/...`).
 2. Make a second API call to write the rich text, using the returned GID as the `data-asana-gid` field of the `<object>` tag. You don't need the inner HTML and you only need a couple of the `<object>` tag attributes: All that is needed is `<object type="application/vnd.asana.external_media" data-asana-gid="..."></object>`
 
+**Blocks and Whitespace**
```

Comment:
This "Blocks and whitespace" section appears twice — above, and here.

Reply:
These considerations apply to both reads (why does my data look like this?) and writes (how do I make my data look like X?). There are other subsections (links, inline images, etc) which show up under both read and write sections, but I agree they are less repetitive. Perhaps I can find a more generalized place to describe the blocks and reference it under the "Read" and "Write" sections.